src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,943 @@
+
+- [Constraint]({% link {{ page.version.version }}/constraints.md %}) violations will abort transactions at all isolation levels.
+
+- In rare cases, a `WriteTooOld` or `RETRY_SERIALIZABLE` error will be returned to the client if a statement has already begun streaming a partial result set back to the client and cannot retry transparently. By default, the result set is buffered up to 16 KiB before overflowing and being streamed to the client. You can configure the result buffer size using the [`sql.defaults.results_buffer.size`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-defaults-results-buffer-size) cluster setting or the [`results_buffer_size`]({% link {{ page.version.version }}/session-variables.md %}) session variable.
+
+### Concurrency anomalies
+
+Statements in concurrent `READ COMMITTED` transactions [can interleave with each other](#write-skew-tolerance). This can create concurrency anomalies that are not permitted under `SERIALIZABLE` isolation, which places concurrent transactions into a [serializable ordering]({% link {{ page.version.version }}/demo-serializable.md %}).
+
+{{site.data.alerts.callout_success}}
+The behavior described in this section assumes the use of non-locking reads. Concurrency anomalies can be prevented through the selective use of [locking reads](#locking-reads), which can also increase latency due to [lock contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention).
+{{site.data.alerts.end}}
+
+#### Write skew anomaly
+
+Because a `READ COMMITTED` transaction's [read timestamp can skew from its write timestamp](#write-skew-tolerance), the following sequence of operations is possible: 
+
+1. Transaction `A` reads row `R` in table `T` at timestamp `1`.
+1. Transaction `B` writes to row `R` in table `T` and commits at timestamp `2`.
+1. Transaction `A` writes to table `T` and commits at timestamp `3`.
```

Comment:
Is this write to a different row or the same row?

Reply:
Writing to a different row -- I guess I should be more explicit about this, since writing to the same row would be a lost update, right? Is there a more general/another way to cause a read refresh to abort a transaction? This is more or less based on the RFC:

> Consequently, it would be possible for a key-value that was read at a transaction's read timestamp to be updated by a second, concurrent transaction at an MVCC timestamp greater than the first transaction's read timestamp but less than the first transaction's commit timestamp. The initial read would "no longer be valid" at the first transaction's commit timestamp. This setup forms the basis for the write skew anomaly.

(Point taken from your other comments -- I'm considering moving all the lower-level behavior to the Transaction Layer architecture doc while I'm here.)